bool is_interrupting
string params
---
